oecologia-australis.csl

```diff
@@ -268,42 +268,54 @@
           <choose>
             <if match="any" variable="number-of-pages publisher">
               <label variable="number-of-pages" form="short" plural="never"/>
-              <text variable="number-of-pages"/>
+              <text variable="number-of-pages" suffix="."/>
               <text variable="publisher" prefix="ed. " suffix=", "/>
               <text variable="publisher-place"/>
             </if>
           </choose>
         </group>
       </if>
       <else-if type="thesis" match="any">
+        <group suffix=".">
```

Comment:
No need for the group here. just do `<text variable="genre" prefix=" " suffix="."/>`

Reply:
I included in a group to be more visualizable during the editions, I think it is not prejudicial.

Reply:
No, sorry, that's just redundant code. Please remove.

Reply:
Ok, I will exclude it.